workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -25,6 +25,17 @@ internal static class DownloadCoreSdk
 
         private const string DownloadForceMenuItem = "SpatialOS/Download CoreSdk (force)";
 
+        private static readonly List<PluginDirectoryCompatibility> pluginsCompatibilityList = new List<PluginDirectoryCompatibility>
```

Comment:
It would be great if there was a way to link this to the paths in `.DownloadCoreSdk.cs`. Otherwise if we change the folder hierarchy we have to change hardcoded paths in 2 files.

Reply:
I would love that as well. The problem is that we deliberately wanted to keep DownloadCoreSDK separated from Unity logic so that it can be run separately (in case Unity bugs with menu item). Considering that, I couldn't come up with a lightweight solution to share the paths (one solution that came to my mind is shared config file, but that one seemed a bit too heavy to just avoid duplicating strings in 2 places).
Happy to hear the suggestions!